src/main/content/start.html

```diff
@@ -90,11 +90,32 @@ <h2 class="section_title">Create a starter application</h2>
     <div class="row">
         <div class="col">
             <div id="starter_warnings"></div>
-            <a id="starter_submit" class="light_green_link_light_background disabled" tabindex="0">Generate Project</a>
+            <a id="starter_submit" class="light_green_link_light_background disabled" tabindex="0" data-toggle="modal">Generate Project</a>
         </div>        
     </div>
 </div>
 
+<!-- Modal popup - Steps to follow when Generate Project Clicked -->
+<div class="modal fade" id="generate-project-modal" role="dialog" aria-labelledby="generate-project-modal-Label" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close generate-project-modal-close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+          </button>
+        </div>
+        <div class="modal-body">
+            <h3>Your zip file has been beamed to your computer!</h3>
+            <p>Unzip it and follow the README instructions or run this command.</p>
+            <div id="cmd_to_run" class="cmd_to_run_class"></div>
```

Comment:
A style suggestion is to remove `_class` from `cmd_to_run` since it is verbose.  Functionally it is fine.

Reply:
done